internal/dbutil/nulltime.go

```diff
@@ -48,3 +49,18 @@ func (nt NullTime) Value() (driver.Value, error) {
 	}
 	return nt.Time, nil
 }
+
+// parseSqliteTimeString parses sqlite times string.
+// It tries to process value as string with timezone first,
+// then fallback to parsing as string without timezone.
+func parseSqliteTimeString(val string) (time.Time, error) {
+	var times time.Time
+	var err error
+
+	times, err = time.Parse(sqliteTimeLayout, val)
+	if err == nil {
+		return times, nil
+	}
+
+	return time.Parse(sqliteTimeLayoutDate, val)
```

Comment:
this makes me really worried. the date in what timezone?

Reply:
If time zone is not provided to `time.Parse` then it is treated as `utc`. As SQLite has no timezone support and we ensure that it we insert `utc` timezone in our code then I guess it should be fine

Reply:
I am little worried about query itself, if we are not deleting `tallies` then I can possibly double count some values. Or maybe this query some performance issue, I wonder if I should lock timestamp and storageusage query it timestamp can possibly be outdated on second connection

Reply:
Is there a way we can always make it UTC in the code that is providing the time?

Reply:
Our tests are checking that queries against sqlite are using UTC. However I think we need an exact description how to handle timezones.